src/findpeaks.jl

```diff
@@ -0,0 +1,131 @@
+module Findpeaks
+
+export findpeaks
+
+"""
+`findpeaks(y::Array{T},
+x::Array{S}=collect(1:length(y))
+;min_height::T=minimum(y), min_prom::T=minimum(y),
+min_dist::S=0, threshold::T=0 ) where {T<:Real,S}`\n
+Returns indices of local maxima (sorted from highest peaks to lowest)
+in 1D array of real numbers. Similar to MATLAB's findpeaks().\n
```

Comment:
Returning the middle index also seems to me to be more reasonable than the first index.
But doing so loses information about the fact that there is a plateau there and, depending on the nature of data, technically not a peak. Also, for non-equally spaced `x`, the "middle" may not be so much a "middle" anymore.

Looking at scipy's [find_peaks](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html) I also noticed they have a `plateau_size` parameter and a corresponding output.
This would be nice to have - then returning the middle index is OK, since there is also output about the size.

If you'll agree I could add this.